sw/device/silicon_creator/mask_rom/BUILD

```diff
@@ -338,12 +338,9 @@ opentitan_functest(
     name = "e2e_bootup_success_otp_dev",
     srcs = ["mask_rom_test.c"],
     cw310 = cw310_params(
-        bitstream = "//hw/bitstream:mask_rom_otp_dev",
+        bitstream = "//hw/bitstream/vivado:fpga_cw310_mask_rom_otp_dev",
         exit_failure = BOOT_FAILURE_MSG,
         exit_success = BOOT_SUCCESS_MSG,
-        # TODO(lowRISC/opentitan#13603): Remove this "manual" tag when the
-        # bitstream target can fetch pre-spliced bitstream from GCP.
-        tags = ["manual"],
```

Comment:
Should this still have the "manual" tag? Did it even need to be in the first place? It looks like the definition of `opentitan_functest()` adds a manual tag [when it calls `native.test_suite()`](https://cs.opensource.google/opentitan/opentitan/+/master:rules/opentitan_test.bzl;l=435;drc=e9c8dc50292c122ac6552c4ba2b62d684826d17f), so I'm not sure deleting it here has any effect.

Reply:
The`manual` tag as applied to the `test_suite` is to prevent the suite from matching a wildcard.

This instance of manual is applied directly to the `e2e_bootup_success_otp_dev_fpga_cw310` rule, which thus also prevents that "subrule" from matching the wildcard (which would get matched by CI, IIRC).

I think this tag should be removed in this PR, as it fixes this test's ability to run in any of the bitstream scenarios (downloaded, created locally, etc).

Reply:
Sounds good!